get-poetry.py

```diff
@@ -931,17 +957,20 @@ def main():
     parser.add_argument(
         "--uninstall", dest="uninstall", action="store_true", default=False
     )
+    parser.add_argument("--file", dest="file", action="store")
```

Comment:
Can we add a help message here as well please? I think the other options are self-explanatory, tthis one might need some context.

Reply:
Added in the latest commit, thanks!